packages/swc-angular/package.json

```diff
@@ -12,6 +12,6 @@
   ],
   "peerDependencies": {
     "@jscutlery/swc-angular-plugin": ">=0.11.0 <1.0.0",
-    "@swc/core": ">=1.4.0 <1.7.0"
+    "@swc/core": ">=1.7.0 <1.10.0"
```

Comment:
version should be bumped due to breaking changes (@swc/core 1.7+ supported only now)